cmake/include_libraries.cmake

```diff
@@ -23,3 +23,4 @@ find_package(SafeC QUIET)
 find_package(Flatbuffers QUIET)
 find_package(ICONV QUIET)
 find_package(UUID QUIET)
+find_package(TIRPC QUIET)
```

Comment:
Trying to find it every time is okay (if wasteful), but it should only try to use it if it can't do it the "normal" way.  It should do a sanity test to make sure it can compile at all and cause a configure error if it cannot.
(Ideal logic:
1. Check if the current usage is functional
2. If not, check for tirpc (fail if not found)
3. Check that tirpc compiles/links/satisfies the current usage (bonus))

Reply:
Sorry, my cmake Kung-fu is obviously non-existent. In order to achieve 1 & 2 above, I need to write a Find cmake script for the default librpc implementation and then conditionally try to find tirpc if we don't find the default? 

If neither pan out do I then just `message( FATAL_ERROR "Unable to locate librpc or libtirpc. Aborting..." )`?

Reply:
I suppose it would be better to re-factor FindTIRPC to FindRPC and just include it all in there. 

Any hints on a shortcut to detect that the default is good since that obviously already works without a find script?